### PR TITLE
More item related decomp

### PR DESCRIPTION
**Pull request by DenSinH**
There would only be one (quite long) function in `items_1.s` left, that I decomped as well as I could so far, but it doesn't quite match. I made the function naked, and put what I had so far behind a `#if 0`. Not sure that is the way to do it, but it would be awkward to have a bunch of files with single functions I thought.

Discovered the meaning of some other fields and functions as well. There are now only 4 (3 long ones in items.s and 1 naked in items_1.c) functions left to decomp regarding items.

### CODE REVIEW

File: src/code_8092334.c [hunk 1]
```diff
@@ -1,15 +1,8 @@
+#include "code_8092334.h"
```
Comment (SethBarberee) [inline src/code_8092334.c:1]: Shouldn't this be under `global.h`?

File: include/code_8092334.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#ifndef GUARD_CODE_8092334_H
+#define GUARD_CODE_8092334_H
+
+#include "global.h"
```
Comment (SethBarberee) [inline include/code_8092334.h:4]: We shouldn't be including `global.h` here.

File: src/code_8092334.c (hunk 1 above)
Comment (DenSinH) [reply src/code_8092334.c:1]: ah whoops, that would fix both problems yeah

**Comment by SethBarberee** [comment]
Is this PR clear to merge?

**Comment by DenSinH** [comment]
alright, I decompiled the last functions left in items.s, so its ready to merge now.

File: src/items.c [hunk 3]
```diff
@@ -76,90 +106,90 @@ bool8 IsThrowableItem(u8 itemIndex)
...
-        param_1->numItems = 0;
-        param_1->unk0 = 0;
+        slot->itemIndex = ITEM_ID_NOTHING;
+        slot->numItems = 0;
+        slot->unk0 = 0;
     }
 }
 
-void sub_8090BB0(struct ItemSlot_ALT *param_1,struct ItemSlot *param_2)
+void SlotToHeltItem(struct HeldItem *held,struct ItemSlot *slot)
```
Comment (SethBarberee) [inline src/items.c:185]:
```suggestion
void SlotToHeldItem(struct HeldItem *held,struct ItemSlot *slot)
```
Or is that name intentional?

**Pull request merged**